frontend/ui/data/creoptions.lua

```diff
@@ -15,6 +15,36 @@ local tableOfNumbersToTableOfStrings = function(numbers)
     return t
 end
 
+-- OS/2 weight classes map
+-- c.f., https://docs.microsoft.com/en-us/typography/opentype/spec/os2#usweightclass
+local usWeightClass = {
+    [100] = C_("Font weight class", "Thin"),
+    [200] = C_("Font weight class", "Extra-light"),
+    [300] = C_("Font weight class", "Light"),
+    [400] = C_("Font weight class", "Regular"),
+    [500] = C_("Font weight class", "Medium"),
+    [600] = C_("Font weight class", "Semi-bold"),
+    [700] = C_("Font weight class", "Bold"),
+    [800] = C_("Font weight class", "Extra-bold"),
+    [900] = C_("Font weight class", "Black"),
+}
+
+local function weightClassToString(weight)
+    if usWeightClass[weight] then
+        return usWeightClass[weight] .. " (" .. weight .. ")"
```

Comment:
Same remark as for the comma. I think we probably already have something fairly silly looking like `%1 (%2)` in there.

Reply:
Mostly have a whole bunch of literals using that syntax pretty much everywhere ^^.

Reply:
Judging by a quick look at strings with parentheses among the translations it may not be an issue.